Makefile

```diff
@@ -44,6 +44,14 @@ test-random: ## Run the application tests in random order
 		xvfb-run $$TEST_CMD ; else \
 		$$TEST_CMD ; fi
 
+FUNCTESTS ?= func_tests
```

Comment:
regarding including the func_tests as part of the make check endpoint - what do you think about the following:

* functional tests are added in `tests/functional/` (nice because it keeps them organized in one place, plus it's what we do on the server side for our functional test suite: https://github.com/freedomofpress/securedrop/tree/develop/securedrop/tests/functional)
* we _can_ if we want add additional makefile targets using pytest's `-k` option to run only the functional tests or only the unit tests though I'm not sure how useful this is (unless pytest-qt is slow), so maybe we just leave out for now since devs can pass options as devs via `TESTOPTS` and we can add later as common patterns arise
* by adding the functional tests to the existing `tests/functional/` folder, it should use the existing `xvfb-run` no? let me know if I misunderstand the constraint here regarding duplicate xvfb runs

Reply:
Ack on moving to `tests/functional`.... it does indeed fix the `xvfb` problem.